lib/manager/components/FeedSourceTableRow.js

```diff
@@ -124,46 +119,69 @@ function relativeDuration (time: any) {
   )
 }
 
+// This method is extracted variable generation which had to be split
+// out of the render function to appease the eslint complexity rule
+function generateFeedSourceLabelRenderVars (feedSource, comparisonValidationSummary) {
```

Comment:
Should types be added to declaration?

Reply:
Absolutely they should. Not sure why flow didn't catch that. Added!